Add News Search API Swagger Spec

xiaojinghe opened the pull request:
* Add News Search API Swagger Spec

* Add NewsArticle into Spec

* Add miss comma before x-ms-examples

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

xiaojinghe committed:
Add News Search API Swagger Spec

* Add News Search API Swagger Spec

* Add NewsArticle into Spec

* Add miss comma before x-ms-examples

olydis commented:
@fearthecowboy FYI, data plane

fearthecowboy commented on specification/cognitiveservices/data-plane/NewsSearch/v1.0/NewsSearch.json:
```diff
@@ -0,0 +1,1121 @@
+{
```
Please switch to 2-space indentation to be consistent with format standards 😄

alvadb reviewed (changes requested):
Please look at the job failures:
linter: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/303479989
model: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/303479990

fearthecowboy commented on specification/cognitiveservices/data-plane/NewsSearch/readme.md:
````diff
@@ -0,0 +1,32 @@
+``` yaml $(tag) == 'release_1_0'
+input-file: v1.0/NewsSearch.json
+```
+
````
Add a section in the `readme.md` that looks like this:

xiaojinghe committed:
suppress _type error, fix 'OBJECT_ADDITIONAL_PROPERTIES'  error, add … (#2)

Suppress _type error, fix 'OBJECT_ADDITIONAL_PROPERTIES'  error, add required parameters

xiaojinghe committed:
Update description and Remove some internal properties (#3)

Update description and Remove some internal properties

AutorestCI commented:
No modification for AutorestCI/azure-sdk-for-ruby

AutorestCI commented:
No modification for AutorestCI/azure-sdk-for-python
